cloudsql/mysql/database-sql/README.md

```diff
@@ -144,31 +164,31 @@ for more details on connecting a Cloud Run service to Cloud SQL.
 gcloud builds submit --tag gcr.io/[YOUR_PROJECT_ID]/run-sql
 ```
 
-2. Deploy the service to Cloud Run:
-
+1. Make sure the following environment variables are set:
 ```sh
-gcloud run deploy run-sql --image gcr.io/[YOUR_PROJECT_ID]/run-sql
+export INSTANCE_CONNECTION_NAME='<MY-PROJECT>:<INSTANCE-REGION>:<INSTANCE-NAME>'
+export DB_USER='<DB_USER_NAME>'
+export DB_PASS='<DB_PASSWORD>'
+export DB_NAME='<DB_NAME>'
 ```
 
-Take note of the URL output at the end of the deployment process.
-
-3. Configure the service for use with Cloud Run
+1. Deploy the service to Cloud Run:
 
 ```sh
-gcloud beta run services update run-sql \
-    --add-cloudsql-instances [INSTANCE_CONNECTION_NAME] \
-    --update-env-vars INSTANCE_CONNECTION_NAME=[INSTANCE_CONNECTION_NAME] \
-    --update-env-vars DB_USER=[YOUR_DB_USER] \
-    --update-env-vars DB_PASS=[YOUR_DB_PASS] \
-    --update-env-vars DB_NAME=[YOUR_DB]
+gcloud run deploy run-sql --image gcr.io/[YOUR_PROJECT_ID]/run-sql \
+  --add-cloudsql-instances $INSTANCE_CONNECTION_NAME \
+  --set-env-vars INSTANCE_CONNECTION_NAME=$INSTANCE_CONNECTION_NAME \
+  --set-env-vars DB_USER=$DB_USER,DB_PASS=$DB_PASS,DB_NAME=$DB_NAME 
```

Comment:
Why the revert in structuring the env vars? --update-env-vars, one per line is preferred.

Also, unclear why export them separately, instead of inline these values.

Reply:
I figured that most users will have these variables exported already, so it would be easiest to just copy-paste the command as-is. I updated it to be inlined in the last commit.